package.json

```diff
@@ -6,14 +6,20 @@
   "scripts": {
     "build-css": "./css-build",
     "check-coverage": "istanbul check-coverage --config .istanbul.yml",
+    "check-unit-coverage": "istanbul check-coverage --config .istanbul.unit.yml",
     "generate-coverage": "yarn run istanbul -- cover _mocha -- --recursive",
-    "git-hook": "yarn run lint && yarn run generate-coverage && yarn run check-coverage && yarn run snyk -- test",
+    "generate-unit-coverage": "yarn run istanbul -- cover _mocha -- --recursive test/unit",
+    "git-hook": "yarn run lint && yarn run generate-unit-coverage && yarn run check-unit-coverage",
+    "snyk-travis": "yarn snyk -- monitor --org=nhsuk",
+    "precommit": "yarn run git-hook",
+    "prepush": "yarn run git-hook && yarn run snyk -- test",
+    "postrewrite": "yarn run git-hook",
     "istanbul": "NODE_ENV=test istanbul",
     "lint": "eslint --ext .js,.json .",
     "start": "node app.js",
     "test": "NODE_ENV=test mocha --recursive test",
     "test-unit": "NODE_ENV=test mocha --recursive test/unit",
-    "travis": "yarn run git-hook && yarn run upload-coverage && yarn run snyk -- test --org=nhsuk && yarn run snyk -- monitor --org=nhsuk",
+    "travis": "yarn run lint && yarn run generate-coverage && yarn run check-coverage && yarn run upload-coverage",
```

Comment:
This could be `yarn git-hook && yarn upload-coverage` to reduce the duplication.